src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -103,9 +103,10 @@
 
 
     @DataBoundConstructor
-    @GeneratePojoBuilder(intoPackage = "*.builder.generated", withFactoryMethod = "*")
```

Comment:
I'm not familiar with this annotation, however I believe this is generating code which might break backward compatibility.

Reply:
Good point. Will leave as is.